### ISSUE TITLE
0.1.12 can no longer be used in Ember component's didInsertElement()

### ISSUE THREAD

**Issue by crusy** [BoxFactura/pulltorefresh.js#35]
0.1.12 introduced an error that makes it unusable in Ember 3.0 (didn't test 3.1 yet). Unfortunately [the commit](https://github.com/BoxFactura/pulltorefresh.js/commit/2c0f182c21f9dcd500ec41afe6517c058c97ca43) is too long to find the proper line, but the effect is

> ember-console.js:29 Unhandled error: TypeError: Cannot read property 'commit' of null
renderer.js:167 Uncaught Error: infinite rendering invalidation detected

Basic example to reproduce (anybody knows how to set up an JSFiddle (or similar) with Ember etc.? Sorry):

```
//app/components/pull-to-refresh.js
import Component from '@ember/component';
import PullToRefresh from 'npm:pulltorefreshjs';

export default Component.extend({

  classNames: ['pull-to-refresh'],

  didInsertElement() {
    this._super(...arguments);

    PullToRefresh.init({
      mainElement: '.pull-to-refresh',
      });
  },
});

// app/templates/some-view.hbs
<div class="some-view">
    {{#pull-to-refresh}}
        ...
    {{/pull-to-refresh}}
</div>
```
The view `some-view` can be opened with v0.1.11, cannot be opened with v0.1.12

**Comment by cbahai** [comment]
`module.exports = PullToRefresh`
add it

**Comment by pateketrueke** [comment]
My apologies, I rebuilt the library not using an UMD format, I fixed and released `v0.1.13` right now!
